### PR TITLE
Added metadata paragraph and example

### PR DESCRIPTION
**Pull request by DavidMacDonald**
to the conformance claim section as per
https://github.com/daisy/epub-revision-a11y/wiki/WCAG-Discovery-Metadata-Proposal

### CODE REVIEW

**Comment by DavidMacDonald** [comment]
Note:  I've put the example inline, however later it might make sense to create a technique with this example, and then just the link will be in the document. This could be considered, but I think it reduce the prominence of the epub example,

File: understanding/20/conformance.html [hunk 1]
```diff
@@ -193,10 +193,42 @@ <h3>Understanding Requirement 5</h3>
...
+    </ul>
+    <p>This page includes both <span property="accessMode" content="textual">text</span>
+       and <span property="accessMode" content="visual">images</span>.
+       <span property="accessibilityFeature" content="alternativeText">Alternative 
+       text</span> is included for all image content and <span 
+         property="accessibilityFeature" content="longDescription">long 
+         descriptions</span> are also provided for images that require more 
+       than simple alternate text. All content is available in text, which 
+       can be accessed by assistive technology.</p>
+</div><code>
```
Comment (alastc) [inline understanding/20/conformance.html]: I assume this should be a closing tag?

Comment (awkawk) [reply understanding/20/conformance.html]: yes, fixed

**Comment by awkawk** [comment]
WG agreed to merge this on May 8 2018. Problematic to merge with Github since the conformance.html page location changed and it was easier to copy and paste the change into the file directly. Closing this PR but the content was merged.

**Pull request closed without merging**